geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportCommand.scala

```diff
@@ -120,9 +120,9 @@ object ExportCommand extends LazyLogging {
     new BufferedOutputStream(compressed)
   }
 
-  def getWriter(params: ExportParams): Writer = new OutputStreamWriter(createOutputStream(params.file, params.gzip))
+  def getWriter[P <: RootExportParams](params: P): Writer = new OutputStreamWriter(createOutputStream(params.file, params.gzip))
```

Comment:
any reason why you can't just pass in params like before and let the inheritance handle this?